app/src/main/java/com/github/gogetters/letsgo/chat/ChatMessage.kt

```diff
@@ -0,0 +1,23 @@
+package com.github.gogetters.letsgo.chat
+
+import java.util.*
+
+class ChatMessage(private val text: String, private val belongsToUser: Boolean, private val sendTime: Date, private val userName: String) {
```

Comment:
They are constant anyway, so I think the Kotlin idiomatic way to do it would be to just make them public. That way we don't have to call getters, and can just access the field (but can't modify it). Also, I think this could be a data class (just add the data keyword)

Reply:
I guess its only needed if we have additional validation logic. Like that you can't set an int to something outside some range